jax/ops/image/util.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+def get_spatial_dim(num_dims, data_format, i):
```

Comment:
I think the (not publicly documented) `lax.conv_dimension_numbers` has most of what you need here. Perhaps we could refactor it a bit and share that code?

Either way I wouldn't want to add a new public API for dimension numbers as part of this change.

Reply:
Removed the util module and changed to use lax's methods.